.changeset/polite-women-dream.md

```diff
@@ -1,5 +0,0 @@
----
-"@solana-program/token-wrap": patch
----
-
-Update CreateMint helper to support token-2022 extension sizing
```

Comment:
That's because we use git cliff on these program repos right? (Changeset is unfortunately JavaScript only).